From 651ca4e4a2d1bb40a48723ddbe575a0229f2575f Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Thu, 10 Jan 2013 13:21:56 +0800
Subject: [PATCH 383/505] ENGR00239567 VPU:Build android test for valid
 platform

This patch changes to build Android vpu unit test only
for valid platform(MX51/53/6DQ/6DL).

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 test/mxc_vpu_test/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/mxc_vpu_test/Android.mk b/test/mxc_vpu_test/Android.mk
index dcafa7f..f5c1f80 100644
--- a/test/mxc_vpu_test/Android.mk
+++ b/test/mxc_vpu_test/Android.mk
@@ -1,3 +1,4 @@
+ifeq ($(BOARD_HAVE_VPU),true)
 LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -23,3 +24,4 @@ LOCAL_SHARED_LIBRARIES := libutils libc libvpu libipu
 LOCAL_MODULE := mxc-vpu-test
 LOCAL_MODULE_TAGS := tests
 include $(BUILD_EXECUTABLE)
+endif
-- 
1.8.0

